package/libs/wolfssl/Config.in

```diff
@@ -16,6 +16,10 @@ config WOLFSSL_HAS_ARC4
 	bool "Include ARC4 support"
 	default y
 
+config WOLFSSL_HAS_CERTGEN
+	bool "Include certificate generation support"
+	default n
```

Comment:
I assume we'd need to set this to `y` in order for px5g-wolfssl to work.
What's the size increase of libwolfssl when doing so (ie. is it less than px5g-standalone...)?

Reply:
The option is automatically selected if pg5x-wolfssl is selected.

Reply:
I don't think this will work on buildbots, we don't have a way for packages to select options of other packages. Please make it default to 'y'.